src/components/App/App.js

```diff
@@ -1,20 +1,34 @@
 import React, { Component } from 'react';
 import Cards from '../Cards/Cards.js';
 import MovieInfo from '../MovieInfo/MovieInfo';
-import movieData from '../../MovieData';
+// import movieData from '../../MovieData';
+import { getMovies, getSingleMovie } from '../../APIFetch'
+
 import './App.css';
 
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
+      movies: [],
+      selectedMovie: null,
       displayMovieInfo: false
     }
   }
 
+  componentDidMount() {
+    getMovies()
+    .then(data => this.setState({movies: data.movies}))
+  }
+
+  showSelectedMovie = (id) => {
```

Comment:
Looking good. We might need to have this in a different component, like MovieSnap or MovieView. We can brainstorm on it.